code/modules/materials/definitions/solids/materials_solid_mineral.dm

```diff
@@ -219,7 +219,7 @@
 	uid = "solid_sand"
 	color = "#e2dbb5"
 	heating_products = list(/decl/material/solid/glass = 1)
-	heating_point = GENERIC_SMELTING_HEAT_POINT
+	heating_point = 2273
```

Comment:
```suggestion
	heating_point = 2000 CELSIUS
```
this is preferred stylistically and also makes it clear why it's 2273 and not 2000